Rakefile

```
@@ -92,6 +96,14 @@ class RepositoryListUpdater
   def repository_url(fqdn, owner, repository)
     "#{owner_url(fqdn, owner)}#{h(repository)}/"
   end
+
+  def mailto_href_subscribe(to)
```

`href` is for `<a>`. This is used for `href` but this is an URL not `href`.

```suggestion
  def subscribe_url(to)
```

(I think that `mailto` is redundant here. Should we include it in method name?)

---

Thank you for the advice.
That's exactly right.
I have fixed this.